build-support/scripts/build-docker.sh

```diff
@@ -1,14 +1,9 @@
-#!/bin/bash
-SCRIPT_NAME="$(basename ${BASH_SOURCE[0]})"
```

Comment:
While tracking down some extraneous stdout/stderr printing I fixed these `pushd/popd` occurrences to be things that didn't modify the current working directory.